src/main/java/roomescape/service/RoomescapeService.java

```diff
@@ -1,28 +1,79 @@
 package roomescape.service;
 
 import java.util.List;
+import java.util.Optional;
 import org.springframework.stereotype.Service;
 import roomescape.domain.Reservation;
+import roomescape.domain.ReservationTime;
+import roomescape.dto.ReservationRequest;
+import roomescape.dto.ReservationResponse;
+import roomescape.dto.ReservationTimeRequest;
+import roomescape.dto.ReservationTimeResponse;
 import roomescape.repository.RoomescapeRepository;
+import roomescape.repository.RoomescapeTimeRepository;
 
 @Service
 public class RoomescapeService {
 
+    public static final int DELETE_FAILED_COUNT = 0;
     private final RoomescapeRepository roomescapeRepository;
```

Comment:
사소할 수 있지만 상수와 상태를 줄바꿈을 통해서 구분하기 쉽게 만들어주면 코드를 파악하기 수월할 것 같아요, 😃

```suggestion
    public static final int DELETE_FAILED_COUNT = 0;
    
    private final RoomescapeRepository roomescapeRepository;
```